core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -441,7 +441,7 @@ class Building : NamedStats(), IConstruction {
     fun isStatRelated(stat: Stat): Boolean {
         if (get(stat) > 0) return true
         if (getStatPercentageBonuses(null).get(stat) > 0) return true
-        if (uniqueObjects.any { it.placeholderText == "[] Per [] Population in this city" && it.stats.get(stat) > 0 }) return true
+        if (uniqueObjects.any { it.placeholderText == "[] Per [] Population []]" && it.stats.get(stat) > 0 }) return true
```

Comment:
Same as above - keep older version, mark with 'deprecated' comment.
Also, you have an extra ] at the end here

Reply:
And lowercase